Datadog/Example/ExampleAppDelegate.swift

```diff
@@ -21,6 +21,12 @@ var rumMonitor: RUMMonitorProtocol { RUMMonitor.shared() }
 class ExampleAppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
+    func logEventMapper(event: LogEvent) -> LogEvent {
+        var event = event
+        event.error?.fingerprint = "mapped"
+        return event
+    }
+
```

Comment:
/question Do we need this in the Example app?

Reply:
No, sorry, I put this in for testing and forgot to remove it.